### PR TITLE
Implement request index page ui top part

### LINKED ISSUE #858: Implement request index page UI top part

### PR DESCRIPTION
**Pull request by prin-r**
Fix #858

### CODE REVIEW

**Comment by sorawit** [comment]
Please fix the conflict now that I landed #990

File: scan/src/rpc/RequestHook.re [hunk 1]
```diff
@@ -91,8 +91,8 @@ module Request = {
            ),
       sufficientValidatorCount: json |> field("sufficientValidatorCount", intstr),
       expirationHeight: json |> field("expirationHeight", intstr),
-      resolveStatus: json |> field("resolveStatus", intstr) |> getResolveStatus,
-      requester: json |> at(["requester"], string) |> Address.fromHex,
+      resolveStatus: Unknown, // TODO , fix this Mock
```
Comment (taobun) [inline scan/src/rpc/RequestHook.re]: Why this line need to change?

**Review by taobun — CHANGES_REQUESTED** [review]

File: scan/src/rpc/RequestHook.re [hunk 2]
```diff
@@ -92,7 +92,7 @@ module Request = {
       sufficientValidatorCount: json |> field("sufficientValidatorCount", intstr),
       expirationHeight: json |> field("expirationHeight", intstr),
       resolveStatus: json |> field("resolveStatus", intstr) |> getResolveStatus,
```
Comment (taobun) [inline scan/src/rpc/RequestHook.re]: resolveStatus: json |> field("resolveStatus", int) |> getResolveStatus,

**Review by taobun — CHANGES_REQUESTED** [review]

File: scan/src/pages/RequestIndexPage.re [hunk 3]
```diff
@@ -20,199 +17,113 @@ module Styles = {
...
+            nowrap=true
+            color=Colors.gray7
+            block=true
+          />
+          <div className=Styles.seperatedLine />
+          {switch (requestOpt) {
+           | Some(request) =>
+             <TimeAgos
+               time={request.requestedAtTime}
+               prefix="Last updated "
```
Comment (evilpeach) [inline scan/src/pages/RequestIndexPage.re]: remove this prefix

File: scan/src/reusable/InfoHL.re [hunk 4]
```diff
@@ -133,6 +136,13 @@ let make = (~info, ~header, ~isLeft=true) => {
           ->Array.of_list
           ->React.array}
        </div>
+     | OracleScript(id, name) =>
+       <div className=Styles.datasourcesContainer>
+         <TypeID.OracleScript id position=TypeID.Subtitle />
+         <HSpacing size=Spacing.sm />
+         <Text value=name size=Text.Lg weight=Text.Regular spacing={Text.Em(0.02)} code=true />
+       </div>
+     | TxHash(txHash, width) => <TxLink txHash width />
```
Comment (evilpeach) [inline scan/src/reusable/InfoHL.re]: font size should be 14px

File: scan/src/reusable/InfoHL.re [hunk 5]
```diff
@@ -152,6 +162,18 @@ let make = (~info, ~header, ~isLeft=true) => {
        <div className={Styles.addressContainer(maxWidth)}>
          <AddressRender address position=AddressRender.Subtitle />
        </div>
+     | Validators(validators) =>
+       <div className=Styles.datasourcesContainer>
+         {validators
+          ->Belt.List.map(validator =>
+              <>
+                <ValidatorMonikerLink validator size=Text.Lg weight=Text.Medium underline=true />
```
Comment (evilpeach) [inline scan/src/reusable/InfoHL.re]: remove `weight=Text.Medium`, use the default value

File: scan/src/pages/RequestIndexPage.re [hunk 6]
```diff
@@ -20,199 +17,113 @@ module Styles = {
...
+           <Col size=2.8>
+             <InfoHL
+               info={
+                 InfoHL.OracleScript(
+                   ID.OracleScript.ID(request.oracleScriptID),
+                   request.oracleScriptName,
+                 )
+               }
+               header="ORACLE SCRIPT"
+             />
```
Comment (evilpeach) [inline scan/src/pages/RequestIndexPage.re:107]: Please set `spacing=Text.Em(0.03)` of header in InfoHL.

**Review by evilpeach — CHANGES_REQUESTED** [review]

File: scan/src/pages/RequestIndexPage.re [hunk 7]
```diff
@@ -1,15 +1,12 @@
 module Styles = {
   open Css;
 
-  let pageContainer = style([paddingTop(`px(50))]);
+  let pageContainer = style([paddingTop(`px(40))]);
```
Comment (evilpeach) [inline scan/src/pages/RequestIndexPage.re]: PaddingTop is too wide. Let's squeeze them to 20px

**Review by evilpeach — CHANGES_REQUESTED** [review]

**Review by evilpeach — APPROVED** [review]
LGTM

**Review by taobun — APPROVED** [review]

**Pull request merged**